package.json

```diff
@@ -48,28 +67,39 @@
     "through2": "^3.0.1",
     "web-encoding": "^1.1.5",
     "xml": "^1.0.0",
-    "xml2js": "^0.4.15"
+    "xml2js": "^0.5.0"
   },
   "devDependencies": {
     "@babel/core": "^7.12.10",
     "@babel/preset-env": "^7.12.10",
+    "@rollup/plugin-babel": "^6.0.3",
+    "@rollup/plugin-node-resolve": "^15.0.2",
+    "@rollup/plugin-replace": "^5.0.2",
+    "@rollup/plugin-typescript": "^11.1.0",
+    "@types/lodash": "^4.14.192",
+    "@types/node": "^18.15.11",
+    "@typescript-eslint/eslint-plugin": "^5.57.1",
+    "@typescript-eslint/parser": "^5.57.1",
     "babelify": "^10.0.0",
     "browserify": "^16.5.2",
     "chai": "^4.2.0",
-    "eslint": "^8.9.0",
-    "gulp": "^4.0.2",
-    "gulp-babel": "^8.0.0",
-    "gulp-eslint": "^4.0.2",
-    "gulp-if": "^3.0.0",
-    "gulp-mocha": "^8.0.0",
-    "gulp-sourcemaps": "^2.6.5",
+    "dotenv": "^16.0.3",
+    "eslint": "^8.37.0",
+    "eslint-config-prettier": "^8.8.0",
+    "eslint-plugin-simple-import-sort": "^10.0.0",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.2.1",
     "mocha": "^9.2.0",
     "mocha-steps": "^1.1.0",
     "nock": "^13.2.2",
+    "prettier": "^2.8.7",
+    "rollup-plugin-node-externals": "^5.1.2",
     "source-map-support": "^0.5.13",
     "split-file": "^2.2.2",
-    "superagent": "^5.1.0",
-    "uuid": "^3.3.2"
+    "superagent": "^8.0.9",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.0.4",
+    "uuid": "^9.0.0"
```

Comment:
we should really remove uuid later on. use `randomUUID` from `node:crypto` instead

Reply:
ah, we're doing CI tests on node 12. 

it's okay then.